torch_geometric/sampler/base.py

```diff
@@ -73,6 +76,19 @@ class BaseSampler(ABC):
         amount of information stored in the sampler, and to initialize all this
         information at `__init__`.
     """
+    @abstractmethod
```

Comment:
Still not a big fan of this. Why do we need it?

Reply:
Agreed, and it's no longer important at all :)